src/service/analyticsService.js

```
@@ -0,0 +1,59 @@
+    }
+  }
+
+  static async getBmiData() {
+    try {
+      const response = await axiosInstance.get(`/analytics/bmi`);
+      return response.data;
+    } catch (error) {
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)